src/pages/uxp-api/reference-js/Modules/uxp/Persistent File Storage/errors.md

```diff
@@ -109,3 +106,11 @@ Unsupported format type
 Data and Format mismatch
 
 
+
+<a name="module-storage-errors-notsupported" id="module-storage-errors-notsupported"></a>
+
+## NotSupported
+Not supported error
```

Comment:
@kareemmyk  We had earlier decided not to show this. Is this needed now?

Reply:
Added @ignore annotation. We can skip it